### PR TITLE
fix problem for newer version

### PR DESCRIPTION
**Pull request by bbuzzanga**
See discussion #226

### CODE REVIEW

**Comment by dbekaert** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/bekaertdavid/ARIA-tools/pullRequest?prid=6192278&bid=20204126)

File: tools/bin/ariaDownload.py [hunk 1]
```diff
@@ -265,8 +264,11 @@ def prod_dl(nt):
     """ Perform downloading using ASF bulk dl; parallel processing supported """
     import multiprocessing
     import ASFDataDload as AD
+    args        = os.sys.argv
```
Comment (dbekaert) [inline tools/bin/ariaDownload.py:267]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=6192279)

**Comment by dbekaert** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=6192279&bid=20204127)

**Comment by rzinke** [comment]
This fixed it for me! Thanks!

File: tools/bin/ariaDownload.py (hunk 1 above)
Comment (dbekaert) [reply tools/bin/ariaDownload.py:267]:
```suggestion
    args = os.sys.argv 
```

**Comment by dbekaert** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=6192279&bid=20204127)

**Comment by dbekaert** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/bekaertdavid/ARIA-tools/pullRequest?prid=6192278&bid=20204126)

File: tools/bin/ariaDownload.py [hunk 2]
```diff
@@ -265,8 +264,11 @@ def prod_dl(nt):
     """ Perform downloading using ASF bulk dl; parallel processing supported """
     import multiprocessing
     import ASFDataDload as AD
+    args = os.sys.argv
```
Comment (dbekaert) [inline tools/bin/ariaDownload.py:267]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/aria-tools/ARIA-tools/pullRequest?prid=6192279)

Comment (dbekaert) [reply tools/bin/ariaDownload.py:267]:
```suggestion
    args = os.sys.argv
```

**Review by dbekaert — APPROVED** [review]
thanks @bbuzz31

**Comment by EJFielding** [comment]
I had to `conda deactivate` and reactivate my conda environment to get the `python setup.py install` to work for me. After that I found this fix worked for me.

**Pull request merged**